data/sql/updates/pending_db_world/rev_1615275106559909076.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1615275106559909076');
+
+-- Reset ranged attack times
+UPDATE `creature_template` SET `RangeAttackTime` = 2000 WHERE `RangeAttackTime` > 0;
```

Comment:
not sure about this, marking as request changes in order to prevent merging this by mistake

Reply:
Very fair, I'm not sure about it either :smile: